packages/core/src/priority-queue/persisted.ts

```diff
@@ -1,6 +1,7 @@
 import { PriorityQueue } from '.'
```

Comment:
This file is not in use FYI, ended up not needing to bring it into core and will remove it when I get a chance.